### DISCUSSION TITLE
npm install

### DISCUSSION THREAD

**Discussion by silgon** [dlemstra/magick-wasm#156] [General]
I did `npm i` and I got:
![image](https://github.com/dlemstra/magick-wasm/assets/901061/5589287c-7ac6-4e03-931a-beb926f6ef5b)
I'm in the most updated `main` branch.
Anyway, the package seems nice btw =)

**Comment by erik-perri** [comment]
You need to setup a personal access token in Github, then login to npm with it in order to install that package from the Github packages registry.

[Docs](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-npm-registry#authenticating-with-a-personal-access-token)

**Reply by silgon** [reply]
Sorry, forgot to reply. Thanks!
